pkg/workload/schemachange/operation_generator.go

```diff
@@ -1791,15 +1791,15 @@ func (og *operationGenerator) dropConstraint(ctx context.Context, tx pgx.Tx) (*o
 		stmt.expectedExecErrors.add(pgcode.FeatureNotSupported)
 	}
 
-	constraintBeingDropped, err := og.constraintInDroppingState(ctx, tx, tableName, constraintName)
+	constraintBeingDropped, err := og.constraintInAddOrDropState(ctx, tx, tableName, constraintName)
 	if err != nil {
 		return nil, err
 	}
 	if constraintBeingDropped {
 		stmt.expectedExecErrors.add(pgcode.FeatureNotSupported)
```

Comment:
We should also expect a `pgcode.UndefinedObject` in the case where we are trying to drop a constraint that doesn't exist

> in the case where we are trying to drop a constraint that doesn't exist

Right now, the workload can try to do something like

```
"previousStatements": [
   "ALTER TABLE schema_w0_39.table_w0_92 DROP CONSTRAINT \"col_w0_92_w1_89_auto_not_null\""
  ]
ERROR: constraint "col_w0_92_w1_89_auto_not_null" of relation "table_w0_92"
does not exist (SQLSTATE 42704)
```

Since at the time of getting this rand constraint, `col_w0_92_w1_89_auto_not_null` existed

Reply:
Sorry the highlight is funky -- I meant to draw emphasis on `stmt.expectedExecErrors.add`

Reply:
ah i see, in that case i'll add that code to the potentialExecErrors